app/logic/emailHandler.py

```diff
@@ -187,3 +189,10 @@ def update_email_template(self):
         except Exception as e:
             print("Error updating email template record: ", e)
             return False
+
+    def retrieve_last_email(event_id):
+        try:
+            get_query = EmailLog.select().where(EmailLog.event==event_id).order_by(EmailLog.dateSent.desc()).get()

```

Comment:
instead of calling this `get_query`, call it what it is. `last_email`, or `last_log`. Or, don't even use a variable and return the query result directly.